lib/core/webhook-request.ts

```diff
@@ -96,7 +96,12 @@ export class WebhookRequest {
     public nodeID (): string {
         let nodeValue: string = null;
         if ("node-id" in this.queryParameters) {
-            nodeValue = this.queryParameters["node-id"];
+            if (typeof this.queryParameters["node-id"] === "string") {
+                nodeValue = this.queryParameters["node-id"] as string;
+            } else {
+                // If parameter have multiple node-id's, we pick first one
+                nodeValue = (this.queryParameters["node-id"] as string[])[0];
```

Comment:
When does this happen?

Reply:
Query parameters like arr=123&arr=321 generates a value arr: [123, 321]. That was why the type in the result of querystring.parse is mappers string or string []. This validation just complies with that in case we get multiple node-id values to match types and avoid the appveyor error (which was correct since the type was not really matching)